### PR TITLE
[api][adm-322] add rate; and fix distance wrong column

### PR DESCRIPTION
**Pull request by jerryfreshworks**
image

### CODE REVIEW

File: api/src/booking/booking.service.ts [hunk 1]
```diff
@@ -255,11 +255,12 @@ export class BookingService {
     setCell({ row: 15, column: 'L', value: booking.interpretFor });
     setCell({ row: 17, column: 'L', value: booking.requestedBy });
 
-    // G83 distance km
-    console.log(distance)
+    // J83 distance km, G83 rate
+    console.info('distance: ', distance);
```
Comment (michel-fresh) [inline api/src/booking/booking.service.ts:259]: maybe remove this log

File: api/src/booking/booking.service.ts [hunk 2]
```diff
@@ -255,11 +255,12 @@ export class BookingService {
...
     if (distance) {
-      setCell({ row: 83, column: 'G', value: distance.distance });
+      setCell({ row: 83, column: 'J', value: distance.distance });
     }
+    setCell({ row: 83, column: 'G', value: 0.55 });
```
Comment (michel-fresh) [inline api/src/booking/booking.service.ts:263]: won't this call twice?

File: api/src/utils/index.ts [hunk 3]
```diff
@@ -195,24 +195,25 @@ export const parseDistanceFromGoogleApi = (data: GoogleDistance) => {
...
 
-  /**
+/**
  * sort array of booking dates by date
  * @param BookingDate[]
  * @param BookingDate[]
  */
 export const sortBookingDates = (a: BookingDateEntity, b: BookingDateEntity) => {
   console.dir(getUnixTime(a.date));
   console.dir(getUnixTime(b.date));
```
Comment (michel-fresh) [inline api/src/utils/index.ts:210]: logalogs

**Review by michel-fresh — COMMENTED** [review]
will look for your reply re line 263 of booking service

File: api/src/booking/booking.service.ts (hunk 1 above)
Comment (jerryfreshworks) [reply api/src/booking/booking.service.ts:259]: ha, Dr. Will wants us to add more logs.

File: api/src/booking/booking.service.ts (hunk 2 above)
Comment (jerryfreshworks) [reply api/src/booking/booking.service.ts:263]: The above one is `J`, this one is `G`

File: api/src/utils/index.ts (hunk 3 above)
Comment (jerryfreshworks) [reply api/src/utils/index.ts:210]: ha, you want me to remove the logs?
saw you are the author.
image

**Pull request merged**